### ISSUE TITLE
Font toggle isn't working in classics editor

### ISSUE THREAD

**Issue by snullp** [writefreely/writefreely#876]
### Describe the bug
Font toggle isn't working in classics editor

### Steps to reproduce (if necessary)
* Choose 'classics' as editor
* Open the editor
* Toggle fonts
* Fonts aren't changing

### Expected behavior
Fonts are set correspondingly

### Application configuration
 - Multi-user
 - **Database?** [mysql]
 - **Open registration?** [yes]
 - **Federation enabled?** [yes]

**Version or last commit**: 0.15

**Comment by elkcityhazard** [comment]
I was able to get the font toggle to work again but I wasn't able to create any content except for the title.

`Uncaught TypeError: $writer.el is null`

I looked at the template and it appears the div containing the ProseMIrror has been removed, and there is code related to prose mirror that has been removed.  

Are you able to actually use the classic editor?

**Comment by snullp** [comment]
Yes I did the same experiments and realized basically the default js selector won't work as the prosa editor loads asynchronously. I couldn't fix it. In the end I did not use the editor because I realized it is not truly WYSIWYG.

**Comment by vtyeh** [comment]
I was able to get this to work ([see here for changes](https://github.com/writefreely/writefreely/compare/develop...vtyeh:writefreely:fix-font-toggle-in-classic-editor-876)) and it involves adding the following fonts to `prose.css`

```
.norm{font-family:Lora,'Palatino Linotype','Book Antiqua','New York','DejaVu serif',serif}
.sans{font-family:'Open Sans','Segoe UI',Tahoma,Arial,sans-serif}
.wrap{font-family:Hack,consolas,Menlo-Regular,Menlo,Monaco,'ubuntu mono',monospace}
```

but i see that we are ignoring css files in the gitignore.
are we able to add the fonts to the prose css?

**Comment by thebaer** [comment]
So all our styles are stored in LESS files. You should be able to modify these (like https://github.com/writefreely/writefreely/blob/develop/less/pad.less), run `make ui` to generate the CSS, and that'll apply to the editor

**Comment by thebaer** [comment]
Thanks @snullp for reporting, and @vtyeh for fixing! This will be in our next release.
